src/main/java/seedu/address/logic/parser/AddCommandParser.java

```diff
@@ -42,9 +41,11 @@ public AddCommand parse(String args) throws ParseException {
         Phone phone = ParserUtil.parsePhone(argMultimap.getValue(PREFIX_PHONE).get());
         Email email = ParserUtil.parseEmail(argMultimap.getValue(PREFIX_EMAIL).get());
         Address address = ParserUtil.parseAddress(argMultimap.getValue(PREFIX_ADDRESS).get());
+        // add command does not allow adding policy straight away. Default is empty arraylist
+        PolicyList policies = new PolicyList(new ArrayList<>());
```

Comment:
Perhaps you could consider moving the `new ArrayList<>()` into the `PolicyList` constructor to better abstract it out